openapi_tester/loaders.py

```diff
@@ -167,6 +169,69 @@ def resolve_path(self, endpoint_path: str) -> tuple:
             )
         raise ValueError(f"Could not resolve path `{endpoint_path}`")
 
+    @staticmethod
```

Comment:
this logic doesnt belong in the loader. the loader just loads the schema.

Reply:
Just saw this now. I also made this change in the #199 PR before I saw this. 

Not sure I 100% get this separation though. The method calls the loader internals and *loads* the response schema section. For the purpose of this package it seems like loading the schema is redundant - the response schema is the unit we're after, and so it feels fine to put in the loaders.py imo.

Nothing in `schema_tester.py` touches this method except `validate_response` which also has access to `self.loader`